### DISCUSSION TITLE
Is it possible to run Steam games on Sandboxie Plus?

### DISCUSSION THREAD

**Discussion by anoNgelo** [sandboxie-plus/Sandboxie#1112] [Q&A Feedback] [answered]
Hi there, 
I am new to sandboxie plus and just started playing around with it, and I just wanted to ask a few questions about gaming on sandboxie.

Q1: Is it possible to run games within the steam app?
     a. If you are able to, can the apps and files within the sandbox scan outside into the host?

Q2: Can you play games that contain anticheats in the sandbox?
     a. If you can, does the anticheat have capabilities to detect that its in a sandbox(if it can detect, will you be banned, or what are the penalties)?
     b. If games with anticheats can be run in sandboxie, can it scan the memory and/or files outside from the host?

Please let me know when you can, and thank you for your time.
P.s. I may come up with more questions about the gaming possibilities on sandboxie for when we start this discussion.

**Comment by isaak654** [comment, marked as answer]
It's possible to run Steam games. However, there are unsolved reports related to not working games that contain anticheats and/or DRM protections.

See the following labels assigned to those reports:
https://github.com/sandboxie-plus/Sandboxie/labels/game%20issue
https://github.com/sandboxie-plus/Sandboxie/labels/launcher%20issue
https://github.com/sandboxie-plus/Sandboxie/labels/DRM%20software
